app/controllers/admin/portfolios_controller.rb

```
@@ -0,0 +1,52 @@
+class Admin::PortfoliosController < AdminsController
+  def add_portfolio
+    portfolio = Portfolio.create(:name        => params[:name],
```

Can you make this a `create!`, otherwise this fails silently.